<launch>

<!-- adding parameters -->
<rosparam param="/moveit/move_group" > "right_arm" </rosparam>
<rosparam param="/filenames/fixed_obstacles" > "/home/ali/catkin_ws/src/yumi_motion_planning/files/fixed_obstacles.txt" </rosparam>
<rosparam param="/filenames/trajs" > "/home/ali/catkin_ws/src/yumi_motion_planning/files" </rosparam>
<rosparam param="/collision_objects/number" > 0 </rosparam>
<rosparam param="/collision_objects/1/dimensions" >[0.095,0.03, 0.045]</rosparam>
<rosparam param="/collision_objects/1/pose" >[0.85, 0.0, 0.0225, 0, 0, 1.57]</rosparam>
<rosparam param="/initial_joint_position/right_arm" >[1.89, -0.996, -1.75,  0.211, -2.15,  1.24,  1.26]</rosparam>
<rosparam param="/target/max_number_of_traj" > 10 </rosparam>
<rosparam param="/target/area_lower_bound" >[0.45,-0.2, 0.17, 3.13, 0.0, 0.0]</rosparam>
<rosparam param="/target/area_upper_bound" >[0.55, 0.2, 0.17, 3.13, 0.0, 0.0]</rosparam>
<rosparam param="/target/steps" >           [0.05, 0.05, 1, 1, 1, 1]</rosparam>
<rosparam param="/target/execute_motion" > 1 </rosparam>


</launch>
